README.md

```diff
@@ -1536,3 +1536,10 @@ ninja
 |        |           | - fix console for rpi5 1.1 rev boards             |
 |        |           | - reorganize the build                            |
 | v0.3.1 | 08/13/2025| add release tag for meta-xt-common                |
+| v0.4.0 | 01/03/2025| - add CAN support for both HATs                   |
```

Comment:
Please, update the section "Features implemented in Release v0.3" in the lines 1484..1488.